### PR TITLE
Adding NO_INSTALL, Updating VTK in CI

### PR DESCRIPTION
**Pull request by mwestphal**
- Adding NO_INSTALL when it makes sense
- Updating VTK commit in CI
- Adding a 9.2.2 CI
- Updating doc about VTK compat

### CODE REVIEW

File: library/VTKExtensions/CMakeLists.txt [hunk 1]
```diff
@@ -9,8 +9,11 @@ vtk_module_scan(
 # if libf3d is built as a static library, VTK extension static libraries
 # must be exported as well for proper linkage
 set(export_name "")
+set(libf3d_vtk_no_install "")
 if(NOT BUILD_SHARED_LIBS)
   set(export_name "f3dTargets")
+else()
+  set(libf3d_vtk_no_install "NO_INSTALL")
```
Comment (Meakk) [inline library/VTKExtensions/CMakeLists.txt:16]: You may want to check the VTK version too.

**Comment by mwestphal** [comment]
https://gitlab.kitware.com/vtk/vtk/-/issues/18677 @Meakk

**Comment by mwestphal** [comment]
please review @Meakk

File: README.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ Available in [Spack](https://spack.readthedocs.io/en/latest/package_list.html#f3
 ## Dependencies
 
 * [CMake](https://cmake.org) >= 3.1.
-* [VTK](https://vtk.org) >= 9.0.0 (optionally with raytracing capabilities to enable OSPray rendering).
+* [VTK](https://vtk.org) >= 9.0.0 (9.2.0 recommended)
```
Comment (mwestphal) [inline README.md:87]: 9.2.2

File: README.md [hunk 3]
```diff
@@ -417,6 +417,10 @@ only the filename or filestem (`.json` will be added) to look for in the locatio
 F3D contains not only the F3D executable but also the libf3d, a library to render 3D meshes, which can be used in C++ or Python.
 The libf3d is documented [here.](README_libf3d.md)
 
+# VTK compatibility
+
+As stated in the dependencies, F3D is compatible with VTK >= 9.0.0, however, many features are only available in certain conditions. We suggest using VTK 9.2.0 with RenderingRayTracing, RenderingExternal and IOExodus modules enabled in order to get as many features as possible in F3D.
```
Comment (mwestphal) [inline README.md:422]: 9.2.2

**Review by Meakk — APPROVED** [review]
Lgtm after your comments are addressed

**Pull request merged**